easybuild/easyconfigs/l/libsigc++/libsigc++-2.10.1-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsigc++'
+version = '2.10.1'
+
+homepage = 'https://libsigcplusplus.github.io/libsigcplusplus/'
+description = """The libsigc++ package implements a typesafe callback system for standard C++."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['c9a25f26178c6cbb147f9904d8c533b5a5c5111a41ac2eb781eb734eea446003']
+
+builddependencies = [
+    ('binutils', '2.30'),
```

Comment:
@jose-d The existing easyconfig also include `Autotools` as a build dependency, which is required (for `M4`), see also #5787, please include it here too.